src/components/github-htmlview.component.js

```diff
@@ -290,6 +297,48 @@ export class GithubHtmlView extends Component {
 
           return undefined;
         },
+        details: (node, index, siblings, parent, defaultRenderer) => {
+          const detailsChildren = node.children || [];
+          const childrendComponent = [];
+          let summaryText = 'details';
+
+          detailsChildren.forEach((child, childIdx) => {
+            if (child.type === 'tag' && child.name === 'summary') {
+              const summaryChild = child.children[0] || {};
+
+              summaryText = summaryChild.data || 'details';
+              summaryText = summaryText.replace(/\n/g, '');
+            } else if (renderers[child.name]) {
+              childrendComponent.push(
+                renderers[child.name](
+                  child,
+                  childIdx,
+                  siblings,
+                  node,
+                  defaultRenderer
+                )
+              );
+            } else {
+              childrendComponent.push(defaultRenderer(child.children, child));
+            }
+          });
+
+          return (
+            <ToggleView
+              TouchableView={collapse => {
+                const prefixNotation = collapse ? '▸' : '▾';
```

Comment:
Those little babies (▸ / ▾) need some style to be bigger.
`prefix` could be a simpler & better variable name